geomesa-hbase/geomesa-hbase-spark-runtime/pom.xml

```diff
@@ -88,6 +88,13 @@
             <artifactId>geomesa-tools_${scala.binary.version}</artifactId>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.protobuf</groupId>
+            <artifactId>protobuf-java</artifactId>
+            <version>2.5.0</version>
+            <scope>runtime</scope>
+        </dependency>
```

Comment:
do we need this change? we should run the cqs script to make sure nothing changed